CONTRIBUTING.md

```diff
@@ -23,6 +26,21 @@ We have sub projects in each branch:
 |   g11n-angular-client   | Singleton Angular 7 client code                                                                                 |
 |   devops                | CI and automation testing code, it is not a place to contribute code unless you want to add CI or test scripts. |
 
+Changes should be made on your own forked branch. PR should be rebased on top of one of above branches without multiple branches mixed into the PR. If your PR do not merge cleanly, use commands listed below to get it up to date.
+
+```
+# origin is the origin Singleton repository
+# upstream is your forked repository under your github account
```

Comment:
Please swich here:
origin should be your forked Singleton
upstream is vmware Singleton